Sources/SwifterSwift/SwiftStdlib/RawRepresentableExtensions.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  RawRepresentableExtensions.swift
+//  SwifterSwift
+//
+//  Created by Guy Kogus on 24/01/2020.
+//  Copyright © 2020 SwifterSwift
+//
+
+// MARK: - Methods (Equatable)
+public extension RawRepresentable where RawValue: Equatable {
+
+    // swiftlint:disable missing_swifterswift_prefix
+
+    /// Returns a Boolean value indicating whether two values are equal.
+    ///
+    /// Equality is the inverse of inequality. For any values `a` and `b`,
+    /// `a == b` implies that `a != b` is `false`.
+    ///
+    /// - Parameters:
+    ///   - lhs: A value to compare.
+    ///   - rhs: Another value to compare.
+    static func == (lhs: Self, rhs: RawValue) -> Bool {
```

Comment:
Do we really need those non-optional ones? If I understand that correctly, we already have an overload of type `(Self, RawValue?) -> Bool` that could handle both optional and non-option params, so maybe if don't need this and let the compiler handle the case by using the `(Self, RawValue?) -> Bool` optional param overload for cases like `Enum.a == "a"`. Unless there is a difference in behavior(which seems not), in that case, we may what to keep this.

Reply:
This comment is also valid for the optional extensions :))

Reply:
Indeed there was no need for this :)